src/start.F90

```diff
@@ -12,10 +12,16 @@ subroutine start
   character(len=128) rmstring
   logical :: outdirthere
 
+  call getarg(1,input_file)
```

Comment:
is there a way to make parameters default if no argument is given?

Reply:
Done. I used command_argument_count which became standard from Fortran 2003 onwards, but it seems to be working fine